be/src/exec/hdfs_scanner.cpp

```diff
@@ -156,6 +206,15 @@ Status HdfsScanner::get_next(RuntimeState* runtime_state, ChunkPtr* chunk) {
             SCOPED_RAW_TIMER(&_app_stats.expr_filter_ns);
             RETURN_IF_ERROR(ExecNode::eval_conjuncts(_scanner_params.conjunct_ctxs, (*chunk).get()));
         }
+
+        if (!_scanner_params.equality_slots.empty()) {
```

Comment:
we'd better to wrap this code into a class? to avoid checking `_scanner_params.quality_slots` all the time. 

Think about use MOR as default mode.